modules/RTC/BridgeChannel.js

```diff
@@ -170,6 +170,20 @@ export default class BridgeChannel {
     }
 
     /**
+     * Sends a "receiver video constraint" message via the channel.
+     * @param {Number} maxFrameHeightPixels the maximum fram height,
+     * in pixels, this receiver is willing to receive
+     */
+    sendReceiverVideoConstraintMessage(maxFrameHeightPixels) {
```

Comment:
i actually think being explicit that this is a receiver constraint is useful (even if we don't ever plan to have a send video constraint message, i still think it's nice to be more descriptive).  do you feel strongly about removing it?